tools/props/Versions.props

```diff
@@ -40,9 +40,11 @@
     <MicrosoftWin32RegistryVersion>5.0.0</MicrosoftWin32RegistryVersion>
     <MicrosoftDataSqlClientSNIRuntimeVersion>5.2.0-preview1.23340.1</MicrosoftDataSqlClientSNIRuntimeVersion>
     <SystemConfigurationConfigurationManagerVersion>6.0.1</SystemConfigurationConfigurationManagerVersion>
+    <SystemConfigurationConfigurationManagerVersionNet8>8.0.0</SystemConfigurationConfigurationManagerVersionNet8>
     <MicrosoftSqlServerServerVersion>1.0.0</MicrosoftSqlServerServerVersion>
     <SystemDiagnosticsPerformanceCounterVersion>6.0.1</SystemDiagnosticsPerformanceCounterVersion>
     <SystemRuntimeCachingVersion>6.0.0</SystemRuntimeCachingVersion>
+    <SystemRuntimeCachingVersionNet8>8.0.0</SystemRuntimeCachingVersionNet8>
```

Comment:
For ease of maintenance, I lean towards keeping a version specific to .NET 6, as it is anticipated to be out of support sooner than .NET 8.

Reply:
@DavoudEshtehari maybe leave as is for now, and then refactor when support for netstandard is removed in 6.0 ?

Reply:
I tried your suggestion, thanks!